Example/src/screens/StackPresentation.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import {ScrollView, StyleSheet, SafeAreaView} from 'react-native';
+import {enableScreens} from 'react-native-screens';
+import {ParamListBase} from '@react-navigation/native';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationProp,
+} from 'react-native-screens/native-stack';
+import {Button} from '../shared';
+
+enableScreens();
+
+const SCREENS = {
+  push: 'push',
```

Comment:
Same as with `stackAnimation`, maybe it is possible to get all options from the type?

Reply:
Unfortunately, [you can't retrieve an array from a type](https://stackoverflow.com/a/59857409)